### PR TITLE
refactoring the hardcoded api url

### PR DESCRIPTION
**Pull request by Yjohn**
refactoring the questionary component and call the API

### CODE REVIEW

File: client/src/containers/PlaceList.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import axios from 'axios';
 import PlaceCard from '../components/Place/PlaceCard';
+import apiClient from '../helpers/apiClient';
```
Comment (Happy0) [inline client/src/containers/PlaceList.js]: It's good that you've separated your API helper methods to use the API into a new file and gave them clear function names :).

Question: Will you have more than one API helper file? Could you perhaps be more specific with the name of the file / variable such as 'questionAPIHandler' to distinguish it from other future API helpers?

(let me know if that doesn't make sense)

File: client/src/helpers/apiClient.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
```
Comment (Happy0) [inline client/src/helpers/apiClient.js]: Good :D

File: client/src/containers/PlaceList.js (hunk 1 above)
Comment (Happy0) [reply client/src/containers/PlaceList.js]: @kabaros What would you prefer style wise? 'questionApiHandler' or 'questionAPIHandler'? :P

Comment (rarmatei) [reply client/src/containers/PlaceList.js]: I think it's supposed to handle both Question and Places APIs so maybe a more general name works?

Comment (kabaros) [reply client/src/containers/PlaceList.js]: yeah it's supposed to handle all API related stuff now - probably the next refactor would be to separate it into questionsApiHandler and anythingElseApiHandler

**Pull request merged**